airflow/executors/kubernetes_executor.py

```diff
@@ -402,22 +405,30 @@ def run_next(self, next_job: KubernetesJobType) -> None:
         status
         """
         self.log.info('Kubernetes job is %s', str(next_job))
+
         key, command, kube_executor_config = next_job
         dag_id, task_id, execution_date, try_number = key
 
-        config_pod = self.worker_configuration.make_pod(
-            namespace=self.namespace,
-            worker_uuid=self.worker_uuid,
-            pod_id=self._create_pod_id(dag_id, task_id),
-            dag_id=self._make_safe_label_value(dag_id),
-            task_id=self._make_safe_label_value(task_id),
+        safe_dag_id = AirflowKubernetesScheduler._make_safe_label_value(dag_id)
+        safe_task_id = AirflowKubernetesScheduler._make_safe_label_value(task_id)
+        safe_date = AirflowKubernetesScheduler._datetime_to_label_safe_datestring(execution_date)
+        safe_pod_id = AirflowKubernetesScheduler._create_pod_id(dag_id, task_id)
+
+        if isinstance(command, str):
+            command = [command]
```

Comment:
Why is this done here? Why is this done at all?

Reply:
@ashb kuberneetes pods require commands be given in a string array. At least that's my guess. @davlum is that the case here?

Reply:
Yeah but when do we set it as a string?

If this was docker a string behaves differntly form an array in `CMD`/`ENTRYPOINT` (one uses the shell, the other doesn't) is all.

Reply:
Honestly I just made this change to satisfy pylint and the recently added ` KubernetesJobType`. I haven't actually confirmed that is actually a string. I'll need to dig into it a bit.